nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-{
-  "name": "Acadia National Park",
-  "params": [
-    "visibility": {
-      "summariesByYear": [
```

Comment:
Each year has its own summary data (I believe these are pulled from the first monitoring site. The front end doesn't care where the data comes from - if it comes in this way, per year, the summary tab will be very easy)

Reply:
Understood.